lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -1605,6 +1607,9 @@ def _get_environment(self):
                 final_env[name] = str(value)
         if self.env:
             for name, value in self.env.items():
+                if not isinstance(value, string_types):
+                    self.fail("Non-string value found for env option. "
+                              "Env options must be wrapped in quotes to avoid YAML parsing. Key: %s Value: %s" % (name, str(value)))
```

Comment:
I would formulate it a bit different (and don't add the value, the key should be fine):
```suggestion
                              "Ambiguous env options must be wrapped in quotes to avoid YAML parsing. Key: %s" % (name, ))
```